### PR TITLE
1.1.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.1.0 (2019-09-20)
+
+- Added demo for the fuzzy datepicker directive and added toggle switches to the datepicker component's demo [#14](https://github.com/blackbaud/skyux2-demos/pull/14)
+- Fixed the vertical tab component's demo to have proper casing. [#12](https://github.com/blackbaud/skyux2-demos/pull/12)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: What about "to use" instead of "to have"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 1.1.0 (2019-09-20)
+
+- Added demo for the fuzzy datepicker directive and added toggle switches to the datepicker component's demo [#14](https://github.com/blackbaud/skyux2-demos/pull/14)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Add a period after demo. :)

Comment (blackbaud-johnly) [reply CHANGELOG.md]: We're also missing the article before "demo." I'd go with "a" instead of "the."

**Review by blackbaud-johnly — APPROVED** [review]
I made the tweak I noticed, plus the 2 Denise pointed out. LGTM.

**Pull request merged**